examples/train_brats2018_new.py

```diff
@@ -19,17 +20,17 @@ def main():
 
     training_generator, val_generator, full_volume, affine = medical_loaders.generate_datasets(args,
                                                                                                path='.././datasets')
-    # model, optimizer = medzoo.create_model(args)
-    # criterion = DiceLoss(classes=args.classes)
-    #
-    # if args.cuda:
-    #     model = model.cuda()
-    #     print("Model transferred in GPU.....")
-    #
-    # trainer = train.Trainer(args, model, criterion, optimizer, train_data_loader=training_generator,
-    #                   valid_data_loader=val_generator)
-    # print("START TRAINING...")
-    # trainer.training()
+    model, optimizer = medzoo.create_model(args)
+    criterion = DiceLoss(classes=args.classes)
+
+    if args.cuda:
+        model = model.cuda()
+        print("Model transferred in GPU.....")
+
+    trainer = train.Trainer(args, model, criterion, optimizer, train_data_loader=training_generator,
+                            valid_data_loader=val_generator)
+    print("START TRAINING...")
```

Comment:
suggestion 2 ( sorrrrryyyyyy): it's better to use logging instead of print

Reply:
we will use logging yeap